pkg/cmd/server/config/priority.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package restore
+package config
```

Comment:
The `priority.go` doesn't depend on any package.
IMO, this file is better placed in the `pkg/util` directory.